sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -99,20 +97,28 @@ def set_workload_id(self, workload_id):
 
     def set_requester_id(self, requester_id):
         """Set requesterId work order parameter."""
+        if requester_id is None or \
+                not is_valid_hex_str(requester_id) or \
+                len(requester_id) != 64:
+            raise ValueError("Empty or Invalid data \
+format for requester_id")
         self.params_obj["requesterId"] = requester_id
 
     def set_worker_encryption_key(self, worker_encryption_key):
         """Set workerEncryptionKey work order parameter."""
-        try:
-            self.params_obj["workerEncryptionKey"] = \
-                worker_encryption_key.encode("UTF-8").hex()
-        except Exception as err:
-            raise TypeError("Worker Encryption Key not valid")
+        if worker_encryption_key is None or \
+                not isinstance(worker_encryption_key, str) or \
+                not worker_encryption_key.strip():
```

Comment:
Why do we need to use `strip()` for worker key. If space character is present then it should be considered as invalid key.

Reply:
Here we are checking if the key contains only space and rejecting the key as a invalid parameter.

Reply:
Can you test removing strip(), anyway you are adding try-catch block for generate_encrypted_key, it will catch the error if it is empty.

Reply:
We don't need strip. If there is a space character in key worker encryption/decryption should fail